src/store/ethers/actions.js

```diff
@@ -102,23 +104,28 @@ export default {
   },
   async freeCredits(ctx) {
 
-    let address = "0x2dad0123e3630535c29911b663cb2a3ef5bae5a1";
-  
-    
-
-    let contract = new Contract(address, ColorBattleContract, getWallet());
-    console.warn(contract)
-    try{
+    // contract address
+    let address = "0x65f9a2dfc14e736e33344bd5f68e88528de58e68";
 
-      let result = await contract.getBalance()
-      console.log("balance", result)
+    // my test rinkeby address
+    let privateKey = "754170781E5F414770E39FE3D1A48060A639C37D1B2A3274CA6C1026FC0CC0BE"
```

Comment:
is this for hard-coding? I think we should be using the provider, the metamask provider?